src/main/java/org/apache/mesos/hdfs/util/DnsResolver.java

```diff
@@ -0,0 +1,96 @@
+package org.apache.mesos.hdfs.util;
+
+import org.apache.commons.logging.Log;
+import org.apache.commons.logging.LogFactory;
+import org.apache.mesos.Protos;
+import org.apache.mesos.SchedulerDriver;
+import org.apache.mesos.hdfs.Scheduler;
+import org.apache.mesos.hdfs.config.SchedulerConf;
+import org.apache.mesos.hdfs.state.PersistentState;
+
+import java.io.IOException;
+import java.net.InetAddress;
+import java.util.HashSet;
+import java.util.Set;
+import java.util.Timer;
+import java.util.TimerTask;
+
+/**
+ * Created by Abhay Agarwal on 3/12/15.
+ */
```

Comment:
What's the point of this comment? The git logs can tell us who created what file when. If we end up with a large enough set of contributors that we need an OWNERS file in the future, then you can be the owner of DnsResolver. Until then, let's use class comments for explaining what the class actually does.